code/modules/client/client procs.dm

```diff
@@ -553,6 +553,19 @@ NOTE:  You will only be polled about this role once per round. To change your ch
 	else
 		view = newView
 
+	if(mob.dark_plane)
+		mob.dark_plane.transform = null
+		var/matrix/M = matrix()
+		M.Scale(view*2.2)
+		mob.dark_plane.transform = M
+
+	if(mob.backdrop)
```

Comment:
This is useless. Literally, this is a backdrop on which you paint objects. Goonlights doesn't work that way.
You need a combination of the flags `KEEP_TOGETHER` and `PLANE_MASTER` for it to have an effect.